src/dawn/engine_dawn_test.cc

```diff
@@ -0,0 +1,44 @@
+// Copyright 2018 The Amber Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "src/dawn/engine_dawn.h"
+
+#include "gmock/gmock.h"
```

Comment:
gtest/gtest.h? Since you aren't mocking anything?

Reply:
Will fix.  I reflexively use gmock.h because I usually use EXPECT_THAT with the more powerful predicates.